packages/angular_devkit/build_angular/src/utils/index-file/augment-index-html.ts

```diff
@@ -171,10 +183,17 @@ export async function augmentIndexHtml(
       linkTags.push(`<link ${attrs.join(' ')}>`);
     }
   }
+  const domainLinkTags: string[] = [];
+  if (imageDomains) {
+    for (const imageDomain of imageDomains) {
+      domainLinkTags.push(`<link rel="preconnect" href="${imageDomain}">`);
```

Comment:
Let's add a "data-" attribute here for tracking?

Reply:
Done!